rust/ql/src/queries/security/CWE-022/TaintedPath.ql

```diff
@@ -0,0 +1,99 @@
+/**
+ * @name Uncontrolled data used in path expression
+ * @description Accessing paths influenced by users can allow an attacker to access unexpected resources.
```

Comment:
Same comment as above.

Reply:
I agree with the comment, but I'm not 100% sold on the suggestion either. I just checked and the 7 other languages use the exact same phrase with the double use of "access". So we should either change all 8 QL and QHelp files consistently or none. 

If you feel strongly about the need to avoid the double use of access then I'd be happy to address this in a separate pull request for all languages. 

I don't really like "reach", it is even more vague than "access" . A clearer phrase would be `... to read, write or modify unexpected resources`, but that is perhaps a bit too long.